apps/builder/app/shared/content-model.test.tsx

```diff
@@ -0,0 +1,356 @@
+import { expect, test } from "vitest";
+import { coreMetas } from "@webstudio-is/sdk";
+import * as baseComponentMetas from "@webstudio-is/sdk-components-react/metas";
+import { $, renderData, ws } from "@webstudio-is/template";
+import { isTreeSatisfyingContentModel } from "./content-model";
+
+// @todo maybe change api to do better errors
```

Comment:
would be great to also add an example of what's wrong right now, later if someone else wants to work on it, they won't know what this means